user.rb

```diff
@@ -0,0 +1,17 @@
+class User
+  attr_accessor :name, :pin, :balance
+
+  def initialize(name, pin, balance)
+    @name    = name
+    @pin     = pin
+    @balance = balance.to_i
```

Comment:
There's not a huge difference, but this is what is generally done to avoid confusion in the future